schema-generic.graphql

```diff
@@ -120,44 +221,71 @@ type FinancialsDailySnapshot @entity {
   timestamp: BigInt!
 }
 
+###########################
+##### Pool-Level Data #####
+###########################
+
 type Pool @entity {
   " Smart contract address of the pool "
   id: ID!
 
   " The protocol this pool belongs to "
   protocol: Protocol!
 
-  " Tokens that need to be deposited to take a position in protocol. e.g. WETH and USDC to deposit into the WETH-USDC pool "
+  " Name of the pool (e.g. Curve.fi DAI/USDC/USDT) "
+  name: String
```

Comment:
Add exclamation mark !!

Reply:
why?

Reply:
not sure if we can assume every pool has a name